sr_external_dependencies/include/sr_external_dependencies/external/common/common_edc_ethercat_protocol.h

```diff
@@ -1,45 +1,26 @@
-//
-// © 2010 Shadow Robot Company Limited.
-//
-// FileName:        common_edc_ethercat_protocol.h
-// Dependencies:
-// Processor:       PIC32
-// Compiler:        MPLAB® C32
-//
-//  +------------------------------------------------------------------------+
-//  | This file is part of The Shadow Robot PIC32 firmware code base.        |
-//  |                                                                        |
-//  | It is free software: you can redistribute it and/or modify             |
-//  | it under the terms of the GNU General Public License as published by   |
-//  | the Free Software Foundation, either version 3 of the License, or      |
-//  | (at your option) any later version.                                    |
-//  |                                                                        |
-//  | It is distributed in the hope that it will be useful,                  |
-//  | but WITHOUT ANY WARRANTY; without even the implied warranty of         |
-//  | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          |
-//  | GNU General Public License for more details.                           |
-//  |                                                                        |
-//  | You should have received a copy of the GNU General Public License      |
-//  | along with this code repository. The text of the license can be found  |
-//  | in Pic32/License/gpl.txt. If not, see <http://www.gnu.org/licenses/>.  |
-//  +------------------------------------------------------------------------+
-//
-//
-//
-//  Doxygen
-//  -------
-//
-//! @file
-//!
-//! The term "Command" means data going from the ROS PC to the Node on the robot
-//! Previously known as "Incoming"
-//!
-//! The term "Status"  means data going from Node on the robot the to the ROS PC
-//! Previously known as "Outgoing"
-//!
-//!
-//! @addtogroup
-//
+/* Copyright 2010, 2023 Shadow Robot Company Ltd.
```

Comment:
Regarding this one, I was actually a bit unsure about what to do, because currently, these files are not exactly the same in both repos.
See for example [here](https://github.com/shadow-robot/hand-firmware/blob/5e902e4bbf2111c04fd7bbfef627daa72e26956c/PIC32/nodes/common/common_edc_ethercat_protocol.h#L48) (hand-fw)
And [here](https://github.com/shadow-robot/sr-ros-interface-ethercat/blob/5cd6388278fb0ebb40a3b58942934808e2e1f905/sr_external_dependencies/include/sr_external_dependencies/external/common/common_edc_ethercat_protocol.h#L64) (sr-ros-interface)

Reply:
What I did, for now, was to copy my changes exactly on both repos. Do you reckon I should copy everything then? (including parts I didn't change)

Reply:
Actually, it seems that all that is left to "copy over" simply changes all references to "SVN revision" to be "Git revision" instead

Reply:
Copied everything (except licence header files for now): https://github.com/shadow-robot/sr-ros-interface-ethercat/pull/265/commits/3b12d452787394119b290cbb2395c916bcdf273a

Reply:
These files are now an exact copy (once this PR is merged: https://github.com/shadow-robot/hand-firmware/pull/34)